### DISCUSSION TITLE
UI elements not visible in "Conditional rules" sidebar

### DISCUSSION THREAD

**Discussion by geertvansoest** [solspace/craft-freeform#471] [Feature Requests]
Hi,

Depending on our screen resolution: when creating a form, some UI elements are not visible in the "Conditional rules" tab in the sidebar when modifying a form field.

For now, we workaround this by zoom-out our browser (to 90%). I did some debugging using the Chrome DevTools, but didn't found out yet which element is causing the problem.

Our resolution: 2560 × 1440 pixels

Kind regard,
Geert van Soest

![Schermafbeelding 2022-09-19 om 16 32 03](https://user-images.githubusercontent.com/6870951/191042471-8ac56740-09df-4dde-a064-ea865171b90e.png)

**Comment by kjmartens** [comment]
Sorry for the trouble, @geertvansoest.

I assume you're on a very small screen or browser window? That's the only way I can see this issue happening. Please let me know. 🙂 

I'll see if there's anything we can do about this, but we are making some large improvements to the form builder experiencing in the near future, and this kind of issue will no longer exist.

**Comment by geertvansoest** [comment]
Hi @kjmartens, some clients and colleagues reported this issue to me. I do self facing this issue on my Macbook (13,3-inch 2560 × 1600) but not on my external monitor (27-inch 2560 × 1440) having my browser window on fullscreen. 

Probably it can be easily fixed for those users by replacing `overflow-x: hidden` to `scroll` on several places.

![image](https://user-images.githubusercontent.com/6870951/194277761-976eda0e-226a-4f88-bdd1-66cd6f40f6e1.png)

**Reply by kjmartens** [reply]
Thanks for this, @geertvansoest. I've made a note about it for a developer to check into soon. 🙂

**Reply by gustavs-gutmanis** [reply]
Hi @geertvansoest,

Could you please upload a full-screen screenshot of the page? (Blur out the sensitive parts)
I would like to see if maybe there's something else causing this issue. I cannot reproduce it on any resolution. I went as high as 5120x1440 and can't reproduce it...

**Reply by geertvansoest** [reply]
Hi @gustavs-gutmanis, sorry my response took so long. A screenshot is attached, which is made on my Macbook with Chrome on fullscreen.

<img width="1440" alt="Schermafbeelding 2022-10-31 om 09 59 57" src="https://user-images.githubusercontent.com/6870951/198971387-58637668-9df4-4fc7-8548-172bab3f80be.png">

As you can see, not all the elements in the sidebar are visible. For example the cross to delete a condition rule.

**Comment by kjmartens** [comment]
The form builder has been overhauled in Freeform 5 for Craft 4 (now in beta), resolving this issue. 🙂
